src/components/EpisodesNav/EpisodesNav.jsx

```diff
@@ -40,24 +45,39 @@ export default function EpisodesNav(){
 	const pagination = data.map(RENDER.pagination);
 
 	return (
-		<nav>
-			<button 
-				className={ s.navButton }
-				aria-label="Previous episode."
-				aria-controls="episodes__items episodes__pagination"
-				aria-hidden={ isFirstEpisodeActive }
-				onClick={ prevEpisode }
-			/>
-			<button 
-				className={ s.navButton }
-				aria-label="Next episode." 
-				aria-controls="episodes__items episodes__pagination"
-				aria-hidden={ isLastEpisodeActive }
-				onClick={ nextEpisode }
-			/>
-			<ol id="episodes__pagination">
-				{ pagination }
-			</ol>
+		<nav className={ s.wrapper }>
+			<div className={ s.container }>
+				<button 
+					className={ `${s.button} ${s.prev}` }
+					aria-label="Previous episode."
+					aria-controls="episodes__items episodes__pagination"
+					aria-hidden={ isFirstEpisodeActive }
+					onClick={ prevEpisode }
+				/>
+				<button 
+					className={ `${s.button} ${s.next}` }
+					aria-label="Next episode." 
+					aria-controls="episodes__items episodes__pagination"
+					aria-hidden={ isLastEpisodeActive }
+					onClick={ nextEpisode }
+				/>
+				{ isSmall && 
+					<ol 
+						className={ s.pagination }
+						id="episodes__pagination"
+					>
+						{ pagination }
+					</ol> 
```

Comment:
It's the same element here and in the other place, so rather than having to update in two places, we should move this into the `renderUtils` I reckon `RENDER.pagination` should refer to this `<ol>`  instead - it can call `renderPaginationLink` to generate its children inside the utils.

Reply:
to clarify, that means we can do
```
const pagination = RENDER.pagination(data);

...

{isSmall && pagination}
...
{!isSmall && pagination}
```

Reply:
Good idea. It didn't sit well with me having two identical lines of code!